api/serializers/andela_centres.py

```diff
@@ -39,8 +39,35 @@ class Meta:
         fields = ("name", "id")
 
 
+class DepartmentDetailSerializer(serializers.ModelSerializer):
+    assets_assigned = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.Department
+        fields = ("name", "id", "assets_assigned")
+
+    def get_assets_assigned(self, obj):
+        """This method returns assets assigned to a particluar department
+
+        Args:
+            obj (object): current object instance being fetched
+
+        Returns:
+            json : serailised assets belonging to the specified department
+        """
+
+        from api.serializers.assets import DepartmentAssetSerializer
+
+        department_assignee = models.AssetAssignee.objects.filter(
+            department_id=obj.id
+        ).first()
+        assets = models.Asset.objects.filter(assigned_to=department_assignee)
+        serialized_assets = DepartmentAssetSerializer(assets, many=True)
```

Comment:
why not use the `AssetSerializer`?

Reply:
The asset serializer calls the department serializer. if I call the asset serializer, it would create a recursive chain.